README.md

```diff
@@ -10,6 +10,7 @@ of the [Privacy Community Group](https://privacycg.github.io/).
 
 - [Kaustubha Govind](https://github.com/krgovind), Google
 - [David Benjamin](https://github.com/davidben), Google
```

Comment:
@HarneetSidhana - could you please remove David Benjamin from the list. He has moved on to other projects, and will not be serving as co-editor for FPS.